store/chat.js

```diff
@@ -115,6 +117,7 @@ export const state = () => ({
   [$states.useIgnore]  : true,
   [$states.trollTts]   : true,
   [$states.ttsRate]    : 10,
+  [$states.ttsTimeout] : 0,
```

Comment:
Should this default to zero? Zero would be instant cancellation on the next tick, so does that make any sense? 
Does zero make more sense than an arbitrary lower limit? 
Should we default to the lower limit? default to upper limit? Middle of the road?

Reply:
0, perhaps unintuitively, disables the limit, which isn't made clear in the UI either.